sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java

```diff
@@ -255,27 +256,40 @@ public class Configs {
     private static final boolean DEFAULT_PARTITION_LEVEL_CIRCUIT_BREAKER_DEFAULT_CONFIG_OPT_IN = false;
     private static final String PARTITION_LEVEL_CIRCUIT_BREAKER_DEFAULT_CONFIG_OPT_IN = "COSMOS.PARTITION_LEVEL_CIRCUIT_BREAKER_DEFAULT_CONFIG_OPT_IN";
 
-    // Config to indicate whether allow insecure connections, for example allow http connection or disable cert verification
+    // Config to indicate whether allow http connections for emulator
     // Please note that this config should only during development or test, please do not use in prod env
-    private static final boolean DEFAULT_INSECURE_EMULATOR_CONNECTION_ALLOWED = false;
-    private static final String INSECURE_EMULATOR_CONNECTION_ALLOWED = "COSMOS.INSECURE_EMULATOR_CONNECTION_ALLOWED";
-    private static final String INSECURE_EMULATOR_CONNECTION_ALLOWED_VARIABLE = "COSMOS_INSECURE_EMULATOR_CONNECTION_ALLOWED";
+    private static final boolean DEFAULT_EMULATOR_HTTP_CONNECTION_ALLOWED = false;
+    private static final String EMULATOR_HTTP_CONNECTION_ALLOWED = "COSMOS.EMULATOR_HTTP_CONNECTION_ALLOWED";
+    private static final String EMULATOR_HTTP_CONNECTION_ALLOWED_VARIABLE = "COSMOS_EMULATOR_HTTP_CONNECTION_ALLOWED";
+
+    // Config to indicate whether disable server certificate validation for emulator
+    // Please note that this config should only during development or test, please do not use in prod env
+    private static final boolean DEFAULT_EMULATOR_CERTIFICATE_VALIDATION_DISABLED = false;
+    private static final String EMULATOR_CERTIFICATE_VALIDATION_DISABLED = "COSMOS.EMULATOR_CERTIFICATE_VALIDATION_DISABLED";
+    private static final String EMULATOR_CERTIFICATE_VALIDATION_DISABLED_VARIABLE = "COSMOS_EMULATOR_CERTIFICATE_VALIDATION_DISABLED";
+
+    // Config to indicate emulator host name
+    // Please note that this config should only during development or test, please do not use in prod env
+    private static final String DEFAULT_EMULATOR_HOST = "localhost";
```

Comment:
I don't think you need the DEFAULT value here (you use multiple variations of localhost hard-coded as default (which is perefctly fine))?

Reply:
yea correct - missed the change here. Thanks